src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -51,16 +70,140 @@ export default function InfoMultipleSection() {
+                            onClick={() => setFilterNodes(!filterNodes)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterNodes')}
+                        </Button>
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterEdges ? 'solid' : 'ghost'}
+                            onClick={() => setFilterEdges(!filterEdges)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterEdges')}
+                        </Button>
+                    </HStack>
+                    {(filterEdges || filterNodes) && (
+                        <Button
+                            m="var(--chakra-space-1)"
+                            width="100%"
+                            size="sm"
+                            onClick={() => setIsOpenChangeModal(true)}
+                            variant="outline"
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.change')}
+                        </Button>
+                    )}
+                    {[...selected]
+                        .filter(id => filterNodes || !(id.startsWith('stn') || id.startsWith('misc_node')))
+                        .filter(id => filterEdges || !id.startsWith('line'))
+                        .map(id => (
+                            <HStack key={id} width="100%">
+                                <Button
+                                    width="100%"
+                                    size="sm"
+                                    variant="solid"
+                                    onClick={() => dispatch(setSelected(new Set([id])))}
+                                    overflow="hidden"
+                                    maxW="270"
+                                    textOverflow="ellipsis"
+                                    whiteSpace="nowrap"
+                                    display="ruby"
+                                >
+                                    {getName(id)?.replaceAll('\\', '⏎')}
+                                </Button>
+                                <Button size="sm" onClick={() => dispatch(removeSelected(id))}>
+                                    <MdClose />
+                                </Button>
+                            </HStack>
+                        ))}
+                </VStack>
+            </Box>
+            <ChangeSelectedModal
+                isOpen={isOpenChangeModal}
+                onClose={() => setIsOpenChangeModal(false)}
+                filterEdges={filterEdges}
+                filterNodes={filterNodes}
+            />
+        </>
+    );
+}
+
+export const ChangeSelectedModal = (props: {
```

Comment:
I find this should be considered as a special case of _Settings - Procedures - Change stations type in batch_ and _Settings - Procedures - Change line styles in batch_. Simply update the procedures modals and use them here should suffice :)